pyanaconda/kickstart.py

```diff
@@ -398,44 +398,6 @@ def setup(self, ksdata):
                 enabled_services.append(NTP_SERVICE)
                 services_proxy.SetEnabledServices(enabled_services)
 
-    def execute(self):
```

Comment:
Just a note, usually, we try to keep the moved code in one commit. Otherwise, it is difficult to find its origin.

Reply:
Fixed...